packages/core/src/Footer/About/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import useStyles from "../useStyles";
+
+import About from "./About";
+import Initiative from "./Initiative";
+
+function Index({ about, initiative, ...props }) {
+  const classes = useStyles(props);
+  return (
+    <div className={classes.root}>
+      <About classes={{ about: classes.about }}>{about}</About>
+      <br />
+      <Initiative classes={{ initiative: classes.initiative }}>
+        {initiative}
+      </Initiative>
```

Comment:
`initiative` is _optional_
```suggestion
      {initiative && (
        <Initiative classes={{ initiative: classes.initiative }}>
          {initiative}
        </Initiative>
      )}
```